projects-yaml-reference_5b6813d20428631d7a89afa9.md

```diff
@@ -0,0 +1,117 @@
+  * [About Projects YAML](#projects-yaml-reference)
+  * [Properties](#properties)
+  * [apiVersion](#apiversion)
+  * [kind](#kind)
+  * [metadata](#metadata)
+    * [name](#name)
+  * [spec](#spec)
+    * [repository](#repository)
+      * [url](#url)
+  * [Example](#example)
+  * [See also](#see-also)
+ 
+# Projects YAML Reference
+
+This document is the reference for the YAML grammar used for adding Semaphore
+2.0 projects to the active Semaphore 2.0 organization using the `create`
+command of the `sem` command line utility.
+
+Projects can also be added to Semaphore 2.0 with the `sem init` command. In
+that case, you will not need to create a YAML file on your own. However,
+`sem init` requires a local copy of the GitHub repository, even if that
+repository is not up to date, that does not contain a
+`.semaphore/semaphore.yml` file. Put simply, `sem init` is simpler but less
+flexible than the `sem create` command.
+
+Note that all branches of a GitHub repository that is added to Semaphore
+2.0 will be visible in Semaphore 2.0 automatically.
```

Comment:
This might be confusing. It will happen only when they make a push to branches.

Reply:
Removed it.